portfolio/src/main/webapp/script.js

```diff
@@ -31,12 +31,71 @@ function getDataComment() {
         const numOfCommentsToDisplay = document.getElementById("numComments").value;
         dataComments.splice(0, numOfCommentsToDisplay).forEach(dataComment => { 
             let miniP = document.createElement("p");
-            miniP.innerText = "Basketball Player: " + dataComment.message;
+            miniP.innerText = dataComment.message + " | Author: " + dataComment.creator;
             commentContainer.appendChild(miniP);
         })
     });
 }
 
+/**
+ * Animates the markers onto the map
+ */
+function toggleBounce() {
```

Comment:
Change all "var" to "const" (or "let" if the value needs to change)

Reply:
Fixed, thanks!